package.json

```diff
@@ -28,11 +28,12 @@
     "build:distfiles": "lerna run --parallel --prefix --concurrency=8 build",
     "build:loose": "run-s -c build:distfiles build:types || true",
     "build:types": "tsc -b",
-    "clean": "run-s clean:*",
-    "clean:distfiles": "npx rimraf \"packages/*/build\"",
+    "clean": "run-s clean:artifacts clean:packages clean:monorepo",
+    "clean:distfiles": "npx rimraf \"packages/*/build/**/*.js\"",
```

Comment:
why are we only getting rid of js and not other things that might be in build? don't we want a totally clean build dir after we run this command?

Reply:
well, because `babel` makes `.js` files and typescript makes `.d.ts` files, and they both go in `build`. neither one depends on the other, so it could be a source of dev-workflow frustration.  `clean:artifacts` removes both.

Reply:
(and `clean:artifacts` is what `clean` calls--these two are more for the case where you just want to clean one or the other)